tests/baselines/reference/callsOnComplexSignatures.types

```diff
@@ -138,7 +138,7 @@ function test4(
 >x : number
 
     arg4: ((a?: {x: number}, b?: {x: number}) => number) | ((a?: {y: number}) => number),
->arg4 : ((a?: { x: number; } | undefined, b?: { x: number; } | undefined) => number) | ((a?: { y: number; } | undefined) => number)
+>arg4 : ((a?: {    x: number;}, b?: {    x: number;}) => number) | ((a?: {    y: number;}) => number)

```

Comment:
This is a little annoying but I guess it’s a general problem with reusing type annotation nodes, not specific to this change 🤔